### ISSUE TITLE
mp4 preference

### ISSUE THREAD

**Issue by xubuntu4iran** [mps-youtube/mps-youtube#71]
https://github.com/np1/mps-youtube/issues/56, after this request was added, I set to 360p; but I think it prefers webm please add an option to prefer mp4

**Comment by cyttorak** [comment]
How can I set mp4 has default option instead of webm?

**Comment by ids1024** [comment]
@cyttorak It hasn't been implemented yet, but probably should. Simple enough, so perhaps in the next release.

**Comment by primaeval** [comment]
A quick hack is to change this line in streams.py
```
 - vo_streams = [x for x in slist if x['mtype'] == "normal" and okres(x)]
 + vo_streams = [x for x in slist if x['mtype'] == "normal" and okres(x) and x['ext'] == "mp4" ]
   
```

**Comment by spvkgn** [comment]
It would be useful if it could be added as setting preferred container with 'set' command.

**Comment by ids1024** [comment]
Implemented in #467
